jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -127,6 +127,28 @@ def get_agent_version(agent_version) {
   }
 }
 
+String generate_gatling_scenario(hours, size, count) {
```

Comment:
Maybe a comment indicating you're going to need to make sure you're in the correct dir or you won't be able to find the corresponding node_configs file because it's a relative path

Reply:
Good call.